server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -319,10 +321,20 @@ private void startStatisticCounts() {
         .getOrCreateAutoGauge(
             Metric.QUANTITY.toString(),
             MetricLevel.IMPORTANT,
-            totalSeriesNumber,
+            totalNormalSeriesNumber,
             AtomicLong::get,
             Tag.NAME.toString(),
-            "timeSeries");
+            "normal-timeSeries");
+
+    MetricsService.getInstance()
+        .getMetricManager()
+        .getOrCreateAutoGauge(
+            Metric.QUANTITY.toString(),
+            MetricLevel.IMPORTANT,
+            totalTemplateSeriesNumber,
+            AtomicLong::get,
+            Tag.NAME.toString(),
+            "template-timeSeries");
```

Comment:
From my perspective, I think these two metrics can use `type` tag to distinguish.
For example,  TAG.NAME.toString(), "timeSeries", TAG.TYPE.toString(), "normal/template"
Besides, please update Metric doc at the same time: 
1. docs/UserGuide/Maintenance-Tools/Metric-Tool.md
2. docs/zh/UserGuide/Maintenance-Tools/Metric-Tool.md

Reply:
Fixed.